src/types/paragraph.d.ts

```diff
@@ -10,9 +10,20 @@ import {
 } from './field_type'
 import { MediaImage } from './media'
 import {
+  NodeBanner,
+  NodeBasicLandingPage,
+  NodeFaqMultipleQA,
+  NodeHealthCareLocalFacility,
   NodeLandingPage,
   NodeNewsStory,
+  NodeOffice,
+  NodePersonProfile,
+  NodePromoBanner,
   NodeQA,
+  NodeRegionalHealthCareServiceDes,
+  NodeStepByStep,
+  NodeStoryListing,
+  NodeSupportResourcesDetailPage,
   NodeSupportService,
 } from './node'
```

Comment:
Why do we have `node` and `media` referenced in paragraph.d.ts? I haven't looked at this very closely but it seems a bit strange

Reply:
Will fix. This happened when merging with main.